src/features/bookOffer/components/BookingInformations.native.test.tsx

```diff
@@ -241,4 +242,59 @@ describe('<BookingInformations />', () => {
 
     expect(screen.queryByText('RUE DE CALI')).not.toBeOnTheScreen()
   })
+
+  it("shouldn't display duration if it has no duration information", () => {
+    // @ts-expect-error mock is not real type
+    mockedUseBookingStock.mockReturnValueOnce({
+      beginningDatetime: '2020-12-01T00:00:00Z',
+    })
+    // @ts-expect-error mock is not real type
+    mockedUseBookingOffer.mockReturnValueOnce({
+      isDigital: false,
+      subcategoryId: SubcategoryIdEnum.CINE_PLEIN_AIR,
+      name: 'Pas de durée',
+      stocks: [],
+      venue: mockOffer.venue,
+      extraData: { durationMinutes: null },
+    })
+    render(<BookingInformations />)
+
+    expect(screen.queryByText('Mardi 1 décembre 2020, 01h00 - Durée : 1h')).not.toBeOnTheScreen()
```

Comment:
Pas trop fan de ce expect il est trop facile à cassé ex:
On change le format d'affichage de l'heure 01h00 devient 1h ce teste sera tout le temps vrai même si l'heur est affiché. je pense qu'il serait mieux d'expect ce que l'on est sensé voir quand on as pas la duration

Reply:
je comprend, j'étais pas à fond non plus. Du coup le test précédent suffit selon toi ?   it("shouldn't display duration if it has no duration information", () => {
    mockedUseBookingStock.mockReturnValueOnce({
      ...offerStockResponseSnap,
      beginningDatetime: randomDatetime,
    })
    mockUseBookingOffer.mockReturnValueOnce({
      ...cinePleinAirOffer,
      extraData: { durationMinutes: null },
    })
    render(<BookingInformations />)

    expect(screen.queryByText('Mardi 1 décembre 2020, 01h00 - Durée : 2h')).not.toBeOnTheScreen()
  })

Reply:
non ici il faut expect le ce que tu devrais avoir quand tu n'as pas la durée